charts/app-config/values-integration.yaml

```diff
@@ -2179,6 +2194,21 @@ govukApplications:
     # TODO: make whitehall-frontend the pattern app and have
     # draft-whitehall-frontend and whitehall-admin inherit from it (instead of
     # the other way around like it is now).
+    ingress:
+      rules:
+        - host: assets-origin.eks.integration.govuk.digital
+          path: /government/placeholder/
+        - host: assets-origin.eks.integration.govuk.digital
+          path: /assets/whitehall/
+        - host: assets-origin.eks.integration.govuk.digital
+          path: /government/uploads/system/uploads/attachment_data/file/*/*/preview
+          pathType: ImplementationSpecific
```

Comment:
Is there a particular reason for specifying `ImplementationSpecific`? Are we overriding it back to the default from something else that's being set elsewhere?

Reply:
If it's intended as self-documentation then that's cool btw - was just curious as to whether its also overriding something or not

Reply:
So the default pathType in the [Helm template](https://github.com/alphagov/govuk-helm-charts/blob/5b6c575fb924e329f4b8ada5b6d2da35be8a842d/charts/generic-govuk-app/templates/ingress.yaml#L29) is `Prefix`. This has specific behaviour to modify the path to prefix path rule in the ALB and prevents you using wildcards (i.e. *) in the path. Need to override it with ImplementationSpecific, so we can write an ALB path rule and use wildcards.

More info here: https://github.com/kubernetes-sigs/aws-load-balancer-controller/pull/1772

Reply:
Ah ok, I did wonder if we were overriding the default somewhere. Thanks.